docs/partner_editable/additional_info.adoc

```diff
@@ -1,5 +1,69 @@
+
+Once there, select *View purchase options* and then choose any of the contract options available. Finally, select *Create contract*. When you are prompted to confirm the purchase, select *Yes*. This will charge your seller account, but you can always request a refund later if the charge is significant.
+
+When the payment is confirmed, select *Set up your account* to be redirected to the registration page. On the registration page, enter your contact information and choose *Submit*. This information, along with your contract entitlement information, will be stored in the AWSMarketplaceSubscribers DynamoDB table.
+
+To verify that testing has passed, confirm the following:
+. The registration page loads fully with no error messages
+. The registration page displays a "Success!" message after you click *Submit*
+. You receive an email notification on the MarketplaceTechAdmin email address with the entitlement information
+. The customer information is stored in the AWSMarketplaceSubscribers DynamoDB table
+    ** To check the DynamoDB table, navigate to DynamoDB in the AWS Management Console, select *Tables* from the navigation menu, choose the *AWSMarketplaceSubscribers* table, and select *Explore table items*. The table should have a new item with the new customer entitlement information.
+
+If all of the above are confirmed, then testing has passed, and your integration is fully operational!
+
+=== Test the Metering (SaaS Subscriptions and Contracts with Consumption)
+
+For SaaS subscriptions and contracts with consumption listings, you need to test that the QuickStart stack is able to correctly call the BatchMeterUsage API to send metering records to the AWS Marketplace Metering Service. To do so, login to AWS under your seller account, and navigate to your AWS Marketplace product listing page.
+
+Once there, select *View purchase options* and then either select a contract option and select *Create contract*, or choose *Subscribe*. When you are prompted to confirm the purchase, select *Yes*. For SaaS contracts with consumption products, this will charge your seller account, but you can always request a refund later if the charge is significant.
+
+When the subscription is confirmed, select *Set up your account* to be redirected to the registration page. On the registration page, enter your contact information and choose *Submit*. This information, along with your subscription information, will be stored in the AWSMarketplaceSubscribers DynamoDB table.
+
+Now that you are subscribed, you need to test that you can send metering records. To do so, follow the steps listed here: https://aws.amazon.com/blogs/awsmarketplace/how-meter-usage-subscriptions-using-saas-quick-start/
+
+To verify that testing has passed, confirm the following:
+. The registration page loads fully with no error messages
+. The registration page displays a "Success!" message after you click *Submit*
```

Comment:
Revise this line to "After you choose **Submit**, the registration page displays a "Success!" message."